provider/datagen/src/driver.rs

```diff
@@ -153,10 +186,17 @@ impl LocaleFamily {
     /// but it does _not_ contain the ancestors "en" and "und".
     ///
     /// Stylized on the CLI as: "%en-US"
+    ///
+    /// The `und` locale is treated specially and behaves like `::single("und")`.
     pub const fn without_ancestors(langid: LanguageIdentifier) -> Self {
```

Comment:
Thought: we previously found that `und` meaning "all locales" was too disruptive / confusing, but `%und` having those semantics _might_ be okay